stdlib/http/post_test.go

```diff
@@ -52,7 +54,7 @@ status = http.post(url:"%s/path/a/b/c", headers: {x:"a",y:"b",z:"c"}, data: byte
 status == 204 or fail()
 `, ts.URL)
 
-	if _, _, err := flux.Eval(script, addFail); err != nil {
+	if _, _, err := flux.Eval(script, context.TODO(), dependencies.NewDefaultDependencies(), addFail); err != nil {
```

Comment:
FYI this one cannot be the test dependencies ATM because it expects a different error code.  This one expects a regular client that will talk to a mock test server, rather than our test dependency setup that uses a mock client.